oaknational/Oak-Web-Application
# Feat/345 blog listing

## Description

- Adds category and author and date to blog items
- Adds pagination component
- Styles blog list and blog list page 
- Adds to test coverage


## CAVEAT: Snippet is currently placeholder lorem ipsum 

Until new field for snippet is worked out in CMS that is just short plain text.

## Issue(s)

Fixes #345 

## How to test

1. Go to https://oak-web-application-git-feat-blog-l-ea21b3-oak-national-academy.vercel.app/blog
2. Click on arrow buttons
3. You should see different listings of blogs

## Checklist
